core/modules/user/user.module

```diff
@@ -1560,11 +1560,11 @@ function user_menu_alter(&$items) {
 }
 
 /**
- * Implements hook_admin_menu_output_alter().
+ * Implements hook_admin_bar_output_alter().
  */
-function user_admin_menu_output_alter(&$content) {
-  // Admin menu shows both the parent menu item and its children, remove the
-  // duplicate child element show "Manage fields" doesn't show up twice.
+function user_admin_bar_output_alter(&$content) {
+  // Admin menu shows both the parent menu item and its children.
+  // Remove the duplicate child element so "Manage fields" doesn't show up twice.
   if (isset($content['menu']['menu']['admin/config']['admin/config/people']['admin/config/people/manage/fields'])) {
```

Comment:
The correct key to check and unset is:
['menu']['menu']['admin/config']['admin/config/people']['admin/config/people/manage']['admin/config/people/manage/fields']

```suggestion
  if (isset($content['menu']['menu']['admin/config']['admin/config/people']['admin/config/people/manage']['admin/config/people/manage/fields'])) {
```